pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -220,9 +225,15 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration, featur
 			if len(kc.EnforceNodeAllocatable) > 1 {
 				allErrors = append(allErrors, fmt.Errorf("invalid configuration: enforceNodeAllocatable (--enforce-node-allocatable) may not contain additional enforcements when %q is specified", kubetypes.NodeAllocatableNoneKey))
 			}
+			continue
```

Comment:
`continue` to skip the following `if !kc.CgroupsPerQOS` check when `EnforceNodeAllocatable` is `[none]`.

Reply:
This is the sort of thing that should be a comment in the code.

```
// skip all further checks when this is explicitly "none"
continue
```

Reply:
Thank you @thockin 
The comment was added now.
Would you mind taking another look?  cc @bart0sh